package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@voltz-protocol/v1-sdk",
-  "version": "1.24.3",
+  "version": "2.0.0",
```

Comment:
This is not proper way to increment major using the semantic-release package we are using. Let's investigate what kind of commit message we need to get this working